app/helper_test.go

```diff
@@ -68,6 +70,11 @@ func setupTestHelper(enterprise bool, tb testing.TB) *TestHelper {
 
 	th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.ListenAddress = prevListenAddress })
 
+	searchEngineBroker, _ := searchengine.NewSearchEngineBroker(th.App.Config(), th.App.Srv.Jobs)
```

Comment:
Same as my other comment. This should be `searchengine.NewBroker`.

Reply:
Good call. I'll do it.